docs/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -52,6 +57,12 @@ Alternatively, you can configure backup store using environment variables:
 - `ZEEBE_BROKER_DATA_BACKUP_S3_REGION` - If no region is provided, it is determined [from the environment](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html#automatically-determine-the-aws-region-from-the-environment).
 - `ZEEBE_BROKER_DATA_BACKUP_S3_ACCESSKEY` - If either `accessKey` or `secretKey` is not provided, the credentials are determined [from the environment](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html#credentials-chain).
 - `ZEEBE_BROKER_DATA_BACKUP_S3_SECRETKEY` - Specify the secret key.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_APICALLTIMEOUT` - Configure a maximum duration for all S3 client API calls.
+- `ZEEBE_BROKER_DATA_BACKUP_S3_FORCEPATHSTYLEACCESS` - When enabled, forces the s3 client to use path-style access. By default, the client will automatically choose between path-style and virtual-hosted-style. Should only be enabled if the s3 compatible storage cannot support virtual-hosted-style.
```

Comment:
As far as casing, I think we want uppercase S3 throughout?

Reply:
Agreed :+1: